src/model/database/db_build.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+const { QueryFile } = require('pg-promise');
+const db = require('./db_connection');
+
+// used to execute a database build file
+const sql = file => QueryFile(path.join(__dirname, file), { minify: true });
```

Comment:
does this parameter `file`need to be in parentheses to satisfy ESlint??

Reply:
actually - this is not an issue - AirB&B only cares with new lines!

Reply:
Don't worry buddy, eslint is ok with it  🍚 👍